srcpkgs/iptables/files/ip6tables/run

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
-[ ! -e /etc/iptables/ip6tables.rules ] && exit 0
-ip6tables-restore -w 3 /etc/iptables/ip6tables.rules || exit 1
+for rule in /etc/iptables/ip6tables.rules /etc/ip6tables.d/*.rules \
+    /etc/ip6tables.d/*.rules ; do
+    [ ! -e "$rule" ] && continue
```

Comment:
`-f`

Reply:
Broken symlinks fail both `-e` and `-f` checks and the previous had `-e` which is why I left it. I'm not married to `-e` (my preference is for `-f`) so updating.